service/static/js-src/muk.institution.js

```diff
@@ -4219,7 +4228,7 @@ $.extend(muk, {
             var inst_buckets = ch.edge.result.buckets("institution");
             for (var i = 0; i < inst_buckets.length; i++) {
                 if (maxSeries > 0 && i >= maxSeries) {
-                    break
+                    return []
```

Comment:
What is reason for returning no data_series in this case? 
I note from lines 4213 and 4222 that maxSeries only ever has values 0 or 10 - is that right?